docs/block-api.md

```diff
@@ -114,6 +114,34 @@ Sometimes a block could have aliases that help users discover it while searching
 keywords: [ __( 'image' ), __( 'photo' ), __( 'pics' ) ],
 ```
 
+#### Styles (optional)
+
+* **Type:** `Array`
+
+Block Styles allow providing alternative styles to block. It works by adding a className to the block’s wrapper. This className can be used to provide an alternative styling for the block if the style variation is selected.
```

Comment:
```suggestion
Block styles can be used to provide alternative styles to block. It works by adding a class name to the block’s wrapper. Using CSS, a theme developer can target the class name for the style variation if it is selected.
```

The specific spelling `className` probably doesn't make much sense to a non-React developer (or someone not closely familiar with JavaScript DOM APIs).